armory/cli/tools/generate_shapes.py

```diff
@@ -0,0 +1,59 @@
+import argparse
+from pathlib import Path
+
+
+def generate_shapes(command_args, prog, description):
+    from armory.utils.shape_gen import (  # move to lazy load to avoid PIL import error
+        Shape,
+    )
```

Comment:
Is there a better way to handle this? In particular, Pillow is listed under `engine` and `develop` but when you install armory-testbed standard it is not included and will error when trying to run `armory --help`. This fixes that problem but creates another that if `generate_shapes` is called at runtime from an environment without PIL the error message will not be useful. However since this is strictly code that is called from either the cli in which the user will know how to fix the issue or from the `carla_adversarial_patch_pytorch` code which must be running with `engine` installed and PIL must also be installed I left it this way.

Reply:
Totally in agreement that all required libraries should be in the `dependencies` section.

Reply:
Totally in agreement that all required libraries should be in the dependencies section.